tests/tck-build-logic/src/main/resources/AllowedDockerImages.txt

```diff
@@ -3,4 +3,5 @@ mariadb:10.8
 postgres:15-alpine
 opengauss/opengauss:3.1.0
 greenmail/standalone:2.0.0
-nginx:1-alpine-slim
\ No newline at end of file
+nginx:1-alpine-slim
+mcr.microsoft.com/mssql/server:2019-CU16-GDR1-ubuntu-20.04
```

Comment:
Can we please use a more recent image? E.g. mcr.microsoft.com/mssql/server:2022-latest has no vulnerabilities rated "high".

Reply:
Can we avoid using `latest` tag? It makes things harder if newer version comes with more vulnerabilities than previous one.

Reply:
In that case we can use 
```
grype mcr.microsoft.com/mssql/server:2019-CU19-ubuntu-20.04
 ✔ Vulnerability DB        [no update available]
 ✔ Pulled image            
 ✔ Loaded image            
 ✔ Parsed image            
 ✔ Cataloged packages      [184 packages]
 ✔ Scanning image...       [68 vulnerabilities]
   ├── 0 critical, 2 high, 38 medium, 13 low, 15 negligible
   └── 38 fixed
```

Reply:
@matneu can you check this image manually?

Reply:
@matneu - I changed the image to `mcr.microsoft.com/mssql/server:2022-RTM-CU2-ubuntu-20.04` since it has no vulnerabilities rated "high"
```
grype mcr.microsoft.com/mssql/server:2022-RTM-CU2-ubuntu-20.04
 ✔ Vulnerability DB        [updated]
 ✔ Pulled image            
 ✔ Loaded image            
 ✔ Parsed image            
 ✔ Cataloged packages      [186 packages]
 ✔ Scanning image...       [53 vulnerabilities]
   ├── 0 critical, 0 high, 21 medium, 13 low, 19 negligible
   └── 14 fixed
```
I will also modify https://github.com/oracle/graalvm-reachability-metadata/pull/245 to create metadata for driver `12.2.0.jre11` instead of `9.4.1.jre16`. The NBT plugin will provide metadata from version 12 when version 9 requested and since those two drivers require exactly the same metadata, everything should work with driver version 9 too.